### PR TITLE
Fix(UI) #7403 : Explore page tabs data loading issue

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the explore page data loading issue form #7403 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

**Before** : 

https://user-images.githubusercontent.com/51777795/190474719-d6d3c943-5879-402f-ac57-ef985f6f5a6a.mov

**After** : 

https://user-images.githubusercontent.com/51777795/190474824-f4fc2e19-4e88-46a1-8912-15fc04d0f264.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx [hunk 1]
```diff
@@ -287,6 +295,116 @@ const Explore: React.FC<ExploreProps> = ({
...
+      }
+    }
+    setIsEntityLoading(false);
+  };
+
+  const fetchData = (value: SearchDataFunctionType[]) => {
+    if (isTourPage) {
+      updateData(mockSearchData as unknown as ExploreSearchData);
+    }
+    const promiseValue = value.map((d) => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx]: Here we should add the else condition if we are on the tour page.

File: openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx [hunk 2]
```diff
@@ -159,15 +155,14 @@ const TourPage = () => {
...
             isFilterSelected
-            error=""
             fetchCount={handleCountChange}
-            fetchData={() => setExploreSearchResult(exploreSearchData)}
             handleFilterChange={handleFilterChange}
             handlePathChange={handleCountChange}
-            handleSearchText={() => setExploreSearchResult(exploreSearchData)}
+            handleSearchText={() => {
+              ('');
+            }}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx:163]: @aniketkatkar97 , Is this correct?

File: openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx [hunk 3]
```diff
@@ -188,55 +134,10 @@ const ExplorePage: FunctionComponent = () => {
...
-      },
-      {
-        queryString: searchText,
-        from: INITIAL_FROM,
-        size: ZERO_SIZE,
-        filters: getFilterString(initialFilter),
-        sortField: initialSortField,
-        sortOrder: INITIAL_SORT_ORDER,
-        searchIndex: getCurrentIndex(tab),
-      },
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx:221]: @aniketkatkar97, As we are removing this from here, are we handling the initial filters on the explore component?

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx]: Done.

File: openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx:163]: No, will make the prop optional here. 👍

File: openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx:221]: These calls were unnecessary as we are doing calls from component level.

**Comment by aniketkatkar97** [comment]
Issue with initial filters reseting after tab change will be handled in other PR.
Created the issue for the same - #7554

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**